pipeline.jsonc

```diff
@@ -4,8 +4,8 @@
         "pipeline-branch": "master",
         "dependencies": // dependencies to pull for a build of contracts, by branch, tag, or commit hash
         {
-            "eosio": "329071ae8ae7c8e467f42079ebdb09e675d7e702", // eventually update to release/2.0.x
-            "eosio.cdt": "aad0c4729b848c9cbe409aa1fdd917797a4d91e7" // eventually update to release/1.7.x
+            "eosio": "develop", 
+            "eosio.cdt": "develop"
```

Comment:
why changed from release to develop? Or. do we have a separated config for release?

Reply:
I don't see why should we fix eos & eosio.cdt branches for fixed commit. so I decided that it will always point to latest develop build.
in master branch this file is overridden after merge though.

Reply:
please double check with Bucky and Jeffery